kitsune/questions/utils.py

```diff
@@ -153,14 +153,21 @@ def flag_question(
     reason: str = FlaggedObject.REASON_CONTENT_MODERATION,
 ) -> None:
     content_type = ContentType.objects.get_for_model(question)
-    FlaggedObject.objects.create(
+    flagged_object, created = FlaggedObject.objects.get_or_create(
         content_type=content_type,
         object_id=question.id,
         creator=by_user,
-        status=status,
-        reason=reason,
-        notes=notes,
+        defaults=dict(
+            reason=reason,
+            status=status,
+            notes=notes,
+        ),
     )
+    if not created:
```

Comment:
This is not needed. You are already using `defaults={}` which will create the values if a CREATE event occurs.